### ISSUE TITLE
Broken links in the gene view page

### ISSUE THREAD

**Issue by pcm32** [ebi-gene-expression-group/atlas-web-core#37]
Orthologs links seem to be broken in the gene view page, for instance, coming from:

https://wwwdev.ebi.ac.uk/gxa/sc/experiments/E-HCAD-4/results/gene-info?geneId=ENSG00000142937&markerGeneK=44&k=44&colourBy=clusters

one faulty gene link would be:
https://wwwdev.ebi.ac.uk/gxa/genes/ENSCING00000006306

```
HTTP Status 404 – Not Found
Type Status Report

Message /gxa/genes/ENSCING00000006306

Description The origin server did not find a current representation for the target resource or is not willing to disclose that one exists.

Apache Tomcat/8.5.31
```

**Comment by alfonsomunozpomer** [comment]
Yes, that’s kind of expected. We’ve never had a great solution for orthologs. That means that the ortholog for chicken has no expression reported in our `Gallus gallus` datasets in bulk.

I remember that we decided to link to bulk by default because it’s where there’s a greater chance of finding ortholog genes, although I guess that a better alternative would be to link only if the gene is present in `bulk-analytics` (maybe we can create a similar collection of `gene2experiment` in bulk to speed this operation, but I’m not sure it’s worth for only a single operation).

Alternatively we could link to Ensembl, but we’d rather keep users in our service, of course. We should discuss this.

**Comment by pcm32** [comment]
Yes, it is tricky, I agree that we want to keep users on our site. Maybe the web application receiving the user at the other side could have a descriptive message saying that there is no expression for that ortholog and offering an automatic redirect back to where they came from? Just an idea of course.

**Comment by alfonsomunozpomer** [comment]
We’ll explore it in the next sprint. There should be some information or obvious message for the user.
